.vscodeignore

```diff
@@ -30,7 +30,7 @@ gulpfile.ts
 !install.Lock
 ISSUE_TEMPLATE
 .mocharc.jsonc
-*.md
+CONTRIBUTING.md
```

Comment:
do we any other md except the changelog?

Reply:
And the readme (though luckily that doesn't get excluded even if we exclude it)

Reply:
Ah I just saw the test-plan as well.   Let me edit this ....